packages/backend/src/Migrations/2025_01_29_admin_upload.ts

```diff
@@ -0,0 +1,31 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+    await db.schema.alterTable('electionDB')
+        /* ballot_source types
+        live_election: Ballots submitted by voters during election
+        prior_election: Election admin uploaded ballots from a previous election
+        */
+        .addColumn('ballot_source', 'varchar' )
```

Comment:
It looks like in the domain model this is a non-optional field, so you should set this to be not null. publick_archive_id is optional and doesn't need this.
Example from other migration:
`.addColumn('election_id', 'varchar', (col) => col.notNull())`

Reply:
Good catch, updated!